addons/website_slides/static/src/js/public/components/slide_share_dialog/email_sharing_input.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="website_slides.EmailSharingInput">
+        <div class="o_wslides_js_share_email">
+            <div t-if="!this.state.isDone and !this.isWebsiteUser" class="input-group">
+                <input t-ref="input" type="text" placeholder="friend1@email.com, friend2@email.com"
+                       class="form-control" t-att-class="{'is-invalid': this.state.isInvalid}"
+                       t-on-keypress="onKeyPress"/>
+                <button type="button" class="btn btn-primary" t-on-click.stop="onShareByEmailClick">
+                    <i class="fa fa-envelope-o mx-1"/>Send Email
+                </button>
+            </div>
+            <div t-if="this.state.isDone or this.isWebsiteUser" class="alert alert-info" role="alert">
+                <span t-if="this.state.isDone"><strong>Sharing is caring!</strong> Email(s) sent.</span>
+                <span t-if="this.isWebsiteUser">Please <a t-attf-href="/web?redirect={{ window.location.href }}" class="fw-bold"> login </a> to share this <t t-esc="this.props.category or 'course'"/> by email.</span>
```

Comment:
Hello :)
I got pinged by the code owner thingy and stumbled on this line with a small oversight.
`'course'` won't be translated here :wink: